webpack/ForemanTasks/Components/TasksTable/TasksTableSchema.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  column,
+  sortableColumn,
+  headerFormatterWithProps,
+  cellFormatter,
+} from 'foremanReact/components/common/table';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { updateURlQuery } from './TasksTableHelpers';
+
+const headFormat = [headerFormatterWithProps];
+const cellFormat = [cellFormatter];
+
+const sortControllerFactory = (apiCall, sortBy, sortOrder) => ({
+  apply: (by, order) => {
+    updateURlQuery({ sort_by: by, sort_order: order });
+    apiCall();
```

Comment:
IMO it should be called `action` because it represents the action you want to run.